src/PrestaShopBundle/Controller/Admin/Sell/Catalog/Product/CombinationController.php

```diff
@@ -276,7 +278,10 @@ public function getListIdsAction(int $productId): JsonResponse
     public function deleteAction(int $combinationId): JsonResponse
     {
         try {
-            $this->getCommandBus()->handle(new DeleteCombinationCommand($combinationId));
+            $this->getCommandBus()->handle(new DeleteCombinationCommand(
+                $combinationId,
+                ShopConstraint::shop($this->getContextShopId()))
```

Comment:
Im not sure how would it look like, it is a whole new concept having shopId in route :thinking:
now route looks like this `/combinations/{combinationId}/delete` so it would become what like `/shop/{shopId}/combinations/{combinationId}/delete` ? On the other hand if we put it in request parameter it would indicate that its kind of optional which is not.
Also if its in route, that means if we generate route in js we need to pass the same "shopId" parameter somewhere in data attribute before, to pass it again :sweat: